src/lumigo_tracer/wrappers/http/http_parser.py

```diff
@@ -170,9 +173,14 @@ def parse_response(self, url: str, status_code: int, headers, body: bytes) -> di
 
 class LambdaParser(ServerlessAWSParser):
     def parse_request(self, parse_params: HttpRequest) -> dict:
+        decoded_uri = safe_split_get(unquote(parse_params.uri), "/", 3)
```

Comment:
Just checked- it can return None (can you update the signature?)
 make sure to handle it in `is_arn`

Reply:
weird - it returns str according to PyCharm

Reply:
you're right. the function's signature is bad. Can you fix it please? :)